src/Client.php

```diff
@@ -1,22 +1,27 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
`strict_types` is probably too risky on this lib - it is better to suppress the relevant sniff in `phpcs.xml.dist` and revert the addition of `strict_types` throughout

Reply:
```xml
    <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing">
        <exclude-pattern>src/</exclude-pattern>
    </rule>
```